components/concept/engine/src/test/java/mozilla/components/concept/engine/prompt/PromptRequestTest.kt

```diff
@@ -256,4 +258,47 @@ class PromptRequestTest {
         assertTrue(onAcceptWasCalled)
         assertTrue(onDismissWasCalled)
     }
+
+    @Test
+    fun `SelectCreditCardPrompt`() {
```

Comment:
nit: Maybe [Given When Then ](https://martinfowler.com/bliki/GivenWhenThen.html)? As we are not really using the backticks as they were meant  to be used.